tests/test_strategies.py

```diff
@@ -198,6 +197,17 @@ def test_ridiculously_parallel_tree_search(self):
             player.tree_search(num_parallel=50)
         self.assertNoPendingVirtualLosses(player.root)
 
+    def test_long_game_tree_search(self):
+        player = MCTSPlayerMixin(DummyNet())
+        player.initialize_game(SEND_TWO_RETURN_ONE)
+        player.root.position.n=108
```

Comment:
can we make this "= MAX_DEPTH - 1" or whatever the relevant relationship is?

Reply:
good idea.  Done.